client/ReviewList.jsx

```diff
@@ -4,14 +4,7 @@ import ReviewItem from './ReviewItem.jsx';
 const ReviewList = props => (
   <div>
     <div>
-      <h4> Reviews </h4>
-    </div>
-    <div>
-      <table>
-        <tbody>
-          {props.reviews.map(review => <ReviewItem review={review} />)}
-        </tbody>
-      </table>
+      {props.reviews.map(review => <ReviewItem review={review} />)}
```

Comment:
same as the comment above, you should use a key for map function

Reply:
I will add a key.  Thanks